docs/framework/network-programming/toc.yml

```diff
@@ -36,6 +36,17 @@
         href: how-to-customize-a-time-based-cache-policy.md
       - name: "How to: Set Cache Policy for a Request"
         href: how-to-set-cache-policy-for-a-request.md
+  - name: FTP
+    href: ftp.md
```

Comment:
Since expander nodes aren't supposed to have associated files, can you put the ftp.md file as a subnode, perhaps with the name "Overview", and add "ftp" as `displayName`?